app/views/student_mailer/finalizing_notification.html.erb

```diff
@@ -5,7 +5,7 @@
   </head>
   <body>
     <p>Hi <%= @user.name %>,</p>
-    <p>Your group is now in the process of finalizing! Please go to <%= "#{@college.site_url} #{@finalizing_path}" %> to confirm your membership.</p>
+    <p>Your group is now in the process of being confirmed. Please go to <%= "#{@college.site_url} #{@finalizing_path}" %> to confirm your membership.</p>
```

Comment:
Won't the space between the url and path cause problems here? Also, I think we can use `<a>` tags for links in the HTML e-mail (e.g. `<a href="<%= @college.site_url + finalizing_path %>Vesta</a>`).